general/device.cpp

```diff
@@ -251,6 +288,28 @@ void Device::Setup(const int device)
    if (Allows(Backend::RAJA_CUDA)) { RajaDeviceSetup(dev, ngpu); }
    // The check for MFEM_USE_OCCA is in the function OccaDeviceSetup().
    if (Allows(Backend::OCCA_MASK)) { OccaDeviceSetup(dev); }
+   if (Allows(Backend::CEED_CPU))
+   {
+      if (!ceed_option)
+      {
+         CeedDeviceSetup("/cpu/self/opt/blocked");
```

Comment:
I just thought of this - if this is set to `/cpu/self/`, then libCEED will pick the best available CPU backend (so LIBXSMM > AVX > Opt > Ref)

Reply:
Thank's that's a really good point!